array.c

```diff
@@ -1016,12 +1016,12 @@ rb_to_array(VALUE ary)
  *  call-seq:
  *    Array.try_convert(object) -> object, new_array, or nil
  *
- *  If +object+ is an +Array+ object, returns +object+.
+ *  Attempts to convert the given +object+ to an +Array+ object:
  *
- *  Otherwise if +object+ responds to <tt>:to_ary</tt>,
- *  calls <tt>object.to_ary</tt> and returns the result.
- *
- *  Returns +nil+ if +object+ does not respond to <tt>:to_ary</tt>
+ *  - If +object+ is an +Array+ object, returns +object+.
+ *  - Otherwise if +object+ responds to <tt>:to_ary</tt>,
+ *    returns <tt>object.to_ary</tt>.
```

Comment:
This may be worth to mention, that a `TypeError` is raised if the result of `object.to_ary` is neither an `Array` nor `nil`.

Reply:
Done.